cg/store/filters/status_case_filters.py

```diff
@@ -200,6 +200,30 @@ def filter_compressible_cases(cases: Query, **kwargs) -> Query:
     return cases.filter(Case.is_compressible)
 
 
+def get_not_received_cases(cases: Query, **kwargs) -> Query:
+    return cases.filter(Sample.received_at == None).distinct()
```

Comment:
Just thinking if we need an extra `select_from(Case)` somewhere.

Reply:
```suggestion
    return Select(cases).filter(Sample.received_at == None).distinct()
```
I think this would be the statement. It would be chained

Reply:
Yeah, just thinking how it works, since the join in the SQL query would return a table with much more columns than the Case table alone does.

Reply:
Nevermind, I think this is a non-issue. But if you in pure SQL write `SELECT * FROM Case c JOIN CaseSample cs ON c.id = cs.case_id`, you would get rows consisting of all the columns in Case and all columns in CaseSample. I was just wondering how it is done in SQLAlchemy